src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspace.cs

```diff
@@ -16,7 +17,7 @@ namespace Microsoft.VisualStudio.LanguageServices
     /// </summary>
     public abstract class VisualStudioWorkspace : Workspace
     {
-        private readonly BackgroundCompiler _backgroundCompiler;
+        private BackgroundCompiler _backgroundCompiler;
```

Comment:
In this case, nulling out the _backgroundCompiler is necessary because we want to change the meaning of PartialSemanticsEnabled. I could have introduced an extra bool field for that, but then we'd have the same discussion about whether that should be readonly. Seemed simpler to save the field and the discussion. (You said it was not  blocking.)